src/nvim/msgpack_rpc/remote_ui.c

```diff
@@ -87,6 +87,7 @@ static Object remote_ui_attach(uint64_t channel_id, uint64_t request_id,
   ui->mouse_on = remote_ui_mouse_on;
   ui->mouse_off = remote_ui_mouse_off;
   ui->insert_mode = remote_ui_insert_mode;
+  ui->replace_mode = remote_ui_insert_mode;
```

Comment:
Should I implement a ```remote_ui_replace_mode``` also?

Can I simply just do a ```push_call(ui, "replace_mode", args);``` and it will be fine?
I did the current "hack" because it doesn't change the current behavior.